<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="system8-PC" ignored="0" name="Groups" tests="6" failures="1" timestamp="04 Sep 2018 08:59:24 GMT" time="0.051" errors="0">
  <testcase name="First" time="0.011" classname="testNG.ReportNg"/>
  <testcase name="second" time="0.011" classname="testNG.ReportNg"/>
  <testcase name="First" time="0.001" classname="testNG.ReportNg1$ReportNg"/>
  <testcase name="third" time="0.019" classname="testNG.ReportNg">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:55)
at junit.framework.Assert.fail(Assert.java:64)
at testNG.ReportNg.third(ReportNg.java:28)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- third -->
  <testcase name="second" time="0.001" classname="testNG.ReportNg1$ReportNg"/>
  <testcase name="third" time="0.001" classname="testNG.ReportNg1$ReportNg"/>
</testsuite> <!-- Groups -->
